docs/components/card.md

```diff
@@ -145,31 +145,27 @@ Set if the card should always have shadow state (shadow)
 
 Cards can be arranged with flex box.
 
-<div class="d-flex flex-row">
+<div class="card-flex-container">
   <KCard
     title="Left"
-    class="w-auto"
     body="This card only has a title"
   />
   <KCard
     title="Center"
-    class="w-auto mx-5"
     body="This card always has a icon button"
   >
     <template v-slot:actions>
       <KButton size="small" appearance="outline">
         <KIcon
-            icon="gearFilled"
-            size="16"
-            view-box="0 0 16 16"
-            class="pr-0"
-          />
+          icon="gearFilled"
+          size="16"
```

Comment:
suggestion: use icon size var

Reply:
This is docs file, it's fine to use hardcoded values here (not in the components though). Same for other places

Reply:
I saw using var in other .md file in this pr

Reply:
Maybe but this PR mostly replaces:
- utility classes (e.g. w-100, mb-4, d-flex, etc.)
- provides hardcoded value fallbacks for color/type/spacing variables and removes functions

I can handle changing icon size in separate PR, it's not in the scope for this one.